src/verify.cc

```diff
@@ -220,6 +237,16 @@ Status verifyJwtWithoutTimeChecking(const Jwt& jwt, const Jwks& jwks) {
         // Verification succeeded.
         return Status::Ok;
       }
+    } else if (jwk->kty_ == "OKP" && jwk->crv_ == "Ed25519") {
```

Comment:
It seems that crv_ = "X25519"  is not used.

Reply:
That's correct. While X25519 keys can be distributed through JWKS they are not used for JWT signing. I added support for them on the JWKS side because it wasn't very much extra work and I figured it was worthwhile for completeness. But if we only care about JWKS supporting things needed for JWT verification then we could remove support for X25519.

Reply:
Then,  let us remove X25519 in JWKS,  otherwise it is confusing.